boost.sh

```
@@ -66,10 +62,7 @@ cd $BUILDDIR/tools/build
 # This is causing havok on different combinations of Ubuntu / Anaconda
 # installations.
 bash bootstrap.sh $TOOLSET
-case $ARCHITECTURE in
-  osx*)  ;;
-  *) export CPLUS_INCLUDE_PATH="$CPLUS_INCLUDE_PATH:$(python3 -c 'import sysconfig; print(sysconfig.get_path("include"))')" ;;
-esac
+export CPLUS_INCLUDE_PATH="$CPLUS_INCLUDE_PATH:$(python3 -c 'import sysconfig; print(sysconfig.get_path("include"))')"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)